src/Hud/Menu/MenuPlugin.cs

```diff
@@ -201,12 +201,12 @@ private void CreateMenu()
             MenuItem xpRateMenu = AddChild(MenuRootButton, "Xph & area", settingsHub.XpRateSettings.Enable, "F10");
             MenuItem areaName = AddChild(xpRateMenu, "Only area name", settingsHub.XpRateSettings.OnlyAreaName);
             AddChild(areaName, "Show latency", settingsHub.XpRateSettings.ShowLatency);
-            AddChild(areaName, "Latency color", settingsHub.XpRateSettings.LatencyTextColor);
+            AddChild(areaName, "Show fps", settingsHub.XpRateSettings.ShowFps);
             AddChild(xpRateMenu, "Show in town", settingsHub.XpRateSettings.ShowInTown);
             AddChild(xpRateMenu, "Font size", settingsHub.XpRateSettings.TextSize);
             AddChild(xpRateMenu, "Fps font color", settingsHub.XpRateSettings.FpsTextColor);
             AddChild(xpRateMenu, "Xph font color", settingsHub.XpRateSettings.XphTextColor);
-            AddChild(xpRateMenu, "Area font color", settingsHub.PreloadAlertSettings.AreaTextColor);
+            AddChild(xpRateMenu, "Area font color", settingsHub.XpRateSettings.AreaTextColor);
```

Comment:
@lapigr In order to get this PR merged, kindly remove all the changes related to XpRateSettings.AreaTextColor.